.github/workflows/go-check.yml

```diff
@@ -79,12 +82,25 @@ jobs:
       - name: go generate
         uses: protocol/multiple-go-modules@v1.2
         if: (success() || failure()) && fromJSON(steps.config.outputs.json).gogenerate == true
+        env:
+          IGNORE_PROTOC_VERSION_COMMENTS: ${{ inputs.go-generate-ignore-protoc-version-comments }}
         with:
           run: |
             git clean -fd # make sure there aren't untracked files / directories
+            if [[ "$IGNORE_PROTOC_VERSION_COMMENTS" == "true" ]]; then
+              find . -name '*.pb.go' -print0 | xargs -0 -r sed -i '/^\/\/.*protoc.*v/d'
+              git add .
+              git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+              git config --global user.name "github-actions[bot]"
```

Comment:
Do I understand correctly that this won't be actually commit the repo? Is this commit only used on CI?

Reply:
Yes, that's 100% right. We're only making a local commit to make it easier to check the diff. But even for local commits, git wants us to set user details.